.gitignore

```diff
@@ -6,4 +6,6 @@ launchSettings.json
 artifacts/
 msbuild.binlog
 .vscode/
-BenchmarkDotNet.Artifacts/
\ No newline at end of file
+BenchmarkDotNet.Artifacts/
+node_modules/
+package-lock.json
```

Comment:
The lock file is not intended to be ignored, we've seen problems with this file wanting to change between different versions of node and windows/non-windows - but this isn't a solution

Reply:
Yeah, I agree. In all other node-based projects, we have the lock file checked in. As it wasn't on Blazor repo, I just made it ignored. When I have the approval for the PR, I'll reset/rebase and push again and will remove the lock file from there and add to the repo.